EightBall.js

```diff
@@ -0,0 +1,39 @@
+/*You've learned a powerful tool in JavaScript: control flow! It's 
+so powerful, in fact, that it can be used to tell someone's fortune.
+
+In this project we will build the Magic Eight Ball using control flow in JavaScript.
+
+The user will be able to input a question, then our program will output a random fortune. */
+
+let userName = 'Karol';
```

Comment:
const userName, nie let, ułatwia czytelność, bo wskazuje na to, że już nigdy tego nie zmieniasz